src/components/asthma/helpers/daily-data-providers/air-quality-home.ts

```diff
@@ -0,0 +1,6 @@
+﻿import { DailyDataQueryResult } from '../../../../helpers/query-daily-data';
+import { collateMaxValueDataPoints, queryAsthmaDeviceData } from './shared';
+
+export default function (startDate: Date, endDate: Date): Promise<DailyDataQueryResult> {
+    return queryAsthmaDeviceData('AirNowApi', 'HomeAirQuality', startDate, endDate, collateMaxValueDataPoints);
```

Comment:
I would move these AirNowApi daily data providers out to the normal daily data providers...this should be able to be reused no

Reply:
Yep, good idea.  I've moved the AQI queries to the the normal daily data providers folder and reworked the Asthma ones a little bit to clear up the confusion we discussed yesterday.